packages/app/src/systems/Transaction/components/ActivityList/ActivityList.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import type { Story } from '@storybook/react';
+
+import { MOCK_TXS } from '../../__mocks__/transactions';
+
+import type { ActivityListProps } from './ActivityList';
+import { ActivityList } from './ActivityList';
+
+import { MOCK_ACCOUNTS } from '~/systems/Account';
+
+export default {
```

Comment:
You can write this story using a `Template` component like Storybook recommend here:

https://storybook.js.org/docs/react/writing-stories/introduction
<img width="744" alt="CleanShot 2023-01-09 at 20 11 42@2x" src="https://user-images.githubusercontent.com/2029172/211426820-20a650ee-2853-4c4a-a83b-a9c866385b4a.png">

And add a `<Box css={{ maxWidth: 300 }}>` to don't have the component in a full-screen window.